runtime/src/internal/manifest-server.d.ts

```diff
@@ -0,0 +1,65 @@
+import { ClientRequest, ServerResponse } from 'http';
+import { TLSSocket } from 'tls';
+import {
+  ComponentModule,
+  SSRComponent
+} from './shared';
+
+export const src_dir: string;
+export const build_dir: string;
+export const dev: boolean;
+export const manifest: Manifest;
+
+export interface Manifest {
+  server_routes: ServerRoute[];
+  ignore: RegExp[];
+  root_comp: ComponentModule
+  error: SSRComponent
+  pages: ManifestPage[]
+}
+
+export interface ManifestPage {
+  pattern: RegExp | null;
+  resources: ManifestResource[]; // list of assets needed for transitive dependencies
```

Comment:
This is only used by my rollup plugin, probably shouldn't be here

Reply:
done